### ISSUE TITLE
ZeroDivisionError 

### ISSUE THREAD

**Issue by sdwilsh** [roflcoopter/viseron#58]
I'm getting this exception:
```
viseron    | [2020-10-11 08:25:38] [lib.camera.front_porch  ] [DEBUG   ] - Running motion detection at 1.0s interval, every 0 frame(s)
viseron    | Exception in thread Thread-7:
viseron    | Traceback (most recent call last):
viseron    |   File "/usr/lib/python3.6/threading.py", line 916, in _bootstrap_inner
viseron    |     self.run()
viseron    |   File "/usr/lib/python3.6/threading.py", line 864, in run
viseron    |     self._target(*self._args, **self._kwargs)
viseron    |   File "/src/viseron/lib/camera.py", line 305, in capture_pipe
viseron    |     if motion_frame_number % motion_decoder_interval_calculated == 0:
viseron    | ZeroDivisionError: integer division or modulo by zero
```

Given [the code](https://github.com/roflcoopter/viseron/blob/8ee3115d4ca794cb488d5eff9236b20b45fa05ce/src/lib/camera.py#L325), it looks like somehow the FPS is being calculated in a way that makes this turn out to be zero.

If I set my `height`, `width`, and `fps` in my config file, I get a slightly more useful error:
```
[2020-10-11 08:33:57] [lib.camera.front_porch] [ERROR   ] - Unable to decode frame. FFMPEG pipe seems broken, message repeated 2 times
```

It looks like a sanity check on the stream needs to happen sooner to provide a more useful message.

**Comment by sdwilsh** [comment]
I realized that my stream was actually broken (I have an allowlist to grant access to the cameras that are on a separate vlan, and I hadn't added this new VM to test out viseron with to it).  Another log message that I saw after adding `height`, `width`, and `fps` to my `config.yaml` gave me the impression that the camera was getting connected to:
```
viseron    | [2020-10-11 08:38:14] [lib.camera.front_porch  ] [ERROR   ] - Restarting frame pipe
viseron    | [2020-10-11 08:38:14] [lib.camera.front_porch  ] [ERROR   ] - Successful reconnection!
```

I know, for a fact, that it could not connect (these errors stopped once I updated my firewall rule).

Would it be possible to make a sanity check on the stream earlier in the process, and provide a human-understandable error message when it fails?
